templates/Default/sidebar.html.twig

```diff
@@ -57,7 +57,7 @@
 
     {# Always visible #}
     <li class="nav-item">
-      <a class="nav-link" href="{{ url('catrobat_web_help') }}" id="btn-tutorials">
+      <a class="nav-link" href="https://wiki.catrobat.org/bin/view/Documentation/" id="btn-tutorials">
```

Comment:
@Tamdav @UX
should the link in the sidebar be opened in a new tab? (similar to footer)

Reply:
I asked Wolfgang and he said it should be opened in the same browser window.